src/instance.rs

```diff
@@ -226,8 +230,39 @@ impl ChorusUser {
                 .as_ref()
                 .map(|info| info.ratelimits.clone()),
             settings,
-            object,
+            object: None,
             gateway,
         }
     }
+
+    /// Sends a request to complete an MFA challenge.
+    /// # Reference
+    /// See <https://docs.discord.sex/authentication#verify-mfa>
+    ///
+    /// If successful, the MFA verification JWT returned is set on the current [ChorusUser] executing the
+    /// request. 
+    ///
+    /// The JWT token expires after 5 minutes.
+    pub async fn complete_mfa_challenge(&mut self, mfa_verify_schema: MfaVerifySchema) -> ChorusResult<()> {
+        let endpoint_url = "/mfa/finish";
```

Comment:
I believe you forgot the api url here:
```suggestion
        let endpoint_url = self.urls.api.clone() + "/mfa/finish";
```

Reply:
Fixed https://github.com/polyphony-chat/chorus/pull/521/commits/ee19cb762fbaa732f06c67cbec1f2f5dace1a80e